src/adaptive_graph_of_thoughts/tests/test_graph_elements.py

```
@@ -0,0 +1,103 @@
+@pytest.fixture
```

Ruff (0.11.9)

Undefined name `pytest`

(F821)